docs/getting-started.adoc

```diff
@@ -1,21 +1,117 @@
 == Getting started
 
-=== Documentation of the original Spring REST Docs
-
-Reference: http://docs.spring.io/spring-restdocs/docs/1.1.2.RELEASE/reference/html5/
-
 === Requirements
 
 Spring Auto REST Docs has the following minimum requirements:
 
 * Java 7
-* Spring Restdocs 1.1.2
+* Spring Restdocs ${spring-restdocs.version} (see http://docs.spring.io/spring-restdocs/docs/${spring-restdocs.version}/reference/html5/[documentation])
 * Jackson has to be used for creating and parsing JSON
 
+=== Usage
+
+. Setup project for http://docs.spring.io/spring-restdocs/docs/${spring-restdocs.version}/reference/html5/#getting-started[Spring REST Docs]
+
+. Configure maven dependency:
++
+[source,xml,indent=0,role="secondary"]
+----
+<dependency>
+    <groupId>org.springframework.restdocs</groupId>
+    <artifactId>spring-restdocs-core</artifactId>
+    <version>${spring-restdocs.version}</version>
+    <classifier>test</classifier>
+    <scope>test</scope>
+</dependency>
+----
+
+. Configure maven javadoc plugin:
++
+[source,xml,indent=0,role="secondary"]
+----
+<build>
+    <plugins>
+        ...
+        <plugin>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <extensions>true</extensions>
+            <executions>
+                <execution>
+                    <phase>compile</phase>
+                    <goals>
+                        <goal>javadoc-no-fork</goal>
+                    </goals>
+                </execution>
+            </executions>
+            <configuration>
+                <doclet>capital.scalable.jsondoclet.ExtractDocumentationAsJsonDoclet</doclet>
+                <docletArtifact>
+                    <groupId>capital.scalable</groupId>
+                    <artifactId>json-doclet</artifactId>
+                    <version>${project.version}</version>
+                </docletArtifact>
+                <destDir>generated-javadoc-json</destDir>
+                <reportOutputDirectory>${project.build.directory}</reportOutputDirectory>
```

Comment:
You have to use esacping here. Otherwise `${project.build.directory}` gets replaced with your build directory.